bin/server.js

```diff
@@ -33,12 +35,17 @@ global.Config.defaults(require('../config/defaults.json'));
 global.Log = Logger.attach(global.Config.get('log:level'), global.Config.get('log:filename'));
 
 // Add request logging middleware
-if (global.Config.get('log:access:level')) {
-  const accessLog = Logger.attach(global.Config.get('log:access:level'), global.Config.get('log:access:filename'));
-
-  app.use(Logger.logRequests((message) => accessLog.log(global.Config.get('log:access:level'), message)));
+if (global.Config.get('log:access')) {
+  deprecate('Separate logging control for access logs has been deprecated and will be removed in a later version.');
 }
 
+app.use(expressWinston.logger({
+  winstonInstance: global.Log,
+  expressFormat: true,
+  level: global.Config.get('log:level'),
```

Comment:
If we want to keep backwards compatibility, where access logs can be logged at a different level, should we set this to `global.Config.get('log:access:level')`?